docs/integrations/index.rst

```diff
@@ -0,0 +1,12 @@
+Integrations
+===============
+
+.. toctree::
+   :maxdepth: 1
+
+   cicd/index
```

Comment:
I think it may be better to hide this toctree with :hidden; and display cards to have something overall disparate

Reply:
It may also fix this 
![image](https://github.com/Giskard-AI/giskard/assets/4953724/306d39a5-d9f7-407e-b1fb-23223edc8b75)
(double integrations displayed)

Reply:
For this, I display now only the child pages (Github, Execute, etc.)